package.json

```diff
@@ -2,23 +2,26 @@
   "author": "JD Huntington <jdh@microsoft.com>",
   "devDependencies": {
     "beachball": "^1.36.2",
-    "lerna": "^3.22.1"
+    "lerna": "^3.22.1",
```

Comment:
Do you still need lerna?

Reply:
I'll refactor it away completely. Good call.